tests/unit/consensus/test_previous_canceled_transaction.py

```
@@ -0,0 +1,74 @@
+
+import pytest
+
+from backend.database_handler.models import TransactionStatus
+from tests.unit.consensus.test_helpers import (
+    consensus_algorithm,  # noqa: F401 - imported so pytest can discover fixture
+    init_dummy_transaction,
+)
+
+
+@pytest.mark.asyncio
+async def test_exec_transaction_continues_after_previous_canceled(consensus_algorithm):
```

Ruff (0.15.2)

[error] Redefinition of unused `consensus_algorithm` from line 7: `consensus_algorithm` redefined here

(F811)